armotypes/postureattackchains.go

```diff
@@ -0,0 +1,57 @@
+package armotypes
+
+type AttackChainStatus string
+type ProcessingStatus string
+
+const (
+	StatusActive AttackChainStatus = "active"
+	StatusFixed  AttackChainStatus = "fixed"
+	// StatusFixedSeen AttackChainStatus = "fixedSeen"
+
+	ProcessingStatusProcessing ProcessingStatus = "processing"
+	ProcessingStatusDone       ProcessingStatus = "done"
+)
+
+type AttackChainType struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+type AttackChain struct {
+	Type             *AttackChainType     `json:"type"`
+	ClusterName      string               `json:"clusterName"`
+	Resource         PortalDesignator     `json:"resource"`
+	AttackChainID    string               `json:"attackChainID"` // name/cluster/resourceID
+	CustomerGUID     string               `json:"customerGUID"`
+	Status           AttackChainStatus    `json:"status"`
```

Comment:
I don't think we need this field currently on the api level, only in the DB structures, no? as we are not returning fixed attack chains in the api